modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -102,10 +82,13 @@ overhead:
     cpu: "500m"
 ----
 
-* The default allocation for virtual machines is 2Gi.
-* The Linux kernel uses approximately 100Mi of memory at boot time.
-* The QEMU process uses approximately 30Mi of memory.
-* The `virtiofsd` process uses approximately 10Mi of memory.
-* The `shim-v2` process uses approximately 20Mi of memory.
+You can change the pod overhead by changing the `spec.overhead` field for a `RuntimeClass`. For example, if the configuration that you run for your containers consumes more than 350Mi of memory for the QEMU process and guest kernel data, you can alter the `RuntimeClass` overhead to suit your needs.
+
+[NOTE]
+====
+The specified default overhead values are supported by Red Hat. Changing default overhead values is not supported and can result in technical issues.
+====
+
+When performing any kind of file system I/O in the guest, file buffers are allocated in the guest kernel. The file buffers are also mapped in the QEMU process on the host, as well as on the `virtiofsd` process.
```

Comment:
@c3d - can you weigh in on this?

Reply:
Good catch. Mapped "in" virtiofsd as well.

The intent is that the memory resource is on the host, it comes from it. It is then mapped into two processes, so it may be counted twice if you are  not careful.